test/avocado/testlib_avocado/seleniumlib.py

```diff
@@ -243,7 +243,7 @@ def send_keys(self, element, text, clear=True, ctrla=False):
 
     def check_box(self, element, checked=True):
         try:
-            if element.get_attribute('checked') != checked:
+            if element.is_selected() != checked:
```

Comment:
Please change the commit message of this commit to be more specific.
When we are changing something in the test directory we usually have the prefix 'test' in the commit message.

Also please avoid using generic words like modifications in the commit tittle, without explaining in the commit message at the actual change.
 
In this case I would like to know why is_selected is prefered over get_attribut('checked')

I am not familiar with selenium testing, so please help me to understand the change for the review.

Reply:
I used that method because i found that the input element has no attribute which is checked, so the method check_box has no expected behavior.And i do some search, it seems that element provide a method whose name is 'is_selected' in selenium, it can get the status of checkbox, so i choose this method.

Reply:
Understood. So in this case the commit message should mention that you fix the wrong method name or something :)